docs/index.rst

```diff
@@ -18,6 +18,7 @@ Contents:
    list_of_services
    common_tasks
    rest_api
+   submitting_pulse_data
```

Comment:
How about a name like 'Submitting via Pulse' or similar?

"Submitting Pulse data" make me think of submitting data, where the data is describing pulse, which is something different :-)